FlowCrypt/src/main/java/com/flowcrypt/email/service/attachment/AttachmentDownloadManagerService.kt

```diff
@@ -63,17 +60,15 @@ import java.util.concurrent.Future
 import javax.mail.Folder
 
 /**
- * This service will be use to download email attachments. To start load an attachment just run service via the intent
+ * This service can be used to download email attachments.
+ * To start loading an attachment just run service via the intent
  * [AttachmentDownloadManagerService.newIntent]
  *
+ * This service provides the following:
  *
- * This service can do:
- *
- *
- *
- *  * Can download simultaneously 3 attachments. The other attachments will be added to the queue.
+ *  * Can download simultaneously 3 attachments. Other attachments will be added to the queue.
  *  * All loading attachments will visible in the status bar
- *  * The user can stop loading an attachment at any time.
+ *  * A user can stop loading an attachment at any time.
```

Comment:
An  user

Reply:
https://english.stackexchange.com/questions/105116/is-it-a-user-or-an-user